lib/actions/createShippingConfirmation.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const fetch = require('node-fetch');
+const requestOptions = require('../requestOptions.js');
+
+exports.process = function(msg, cfg) {
+  let self = this;
+  let orderId = parseInt(msg.body.vms.updateOrderStatus.data.id);
+  let carrierServiceName = msg.body.webhook.CarrierServiceName;
+  let trackingId = msg.body.webhook.ConsignmentTrackingReference;
+  let consignmentState = msg.body.webhook.ConsignmentState;
+
+  function createShippingConfirmation(authenticationHeaders) {
+    if (consignmentState != "Manifested") {
+      console.log(`Consignment status is ${consignmentState}, skipping creation of shipping confirmation.`);
```

Comment:
We are not console logging the data returned from the promise though. The message in the promise is there because:
a) we need to return something even when the action did nothing so that we can see the little green counter in the UI
b) it is more informative that just `true` or `false` in case someone tries to chain something to this action in the future.